operations/customers.md

```diff
@@ -0,0 +1,169 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# Customers
+
+## Get customers
+
+This operation returns a list of customers
+
+**Note:** This operation needs [Authentication](../guidelines/authentication.md) and supports the following JSON:API features:
+
+- [Sparse fieldsets](../guidelines/sparse-fieldsets.md) - supports all fields of `customer` query parameter.
+
+### Request
+
+`GET` `[PlatformAddress]/api/v2/customers`
+
+### Response
+
+```javascript
+{
+  "data": [
+    {
+      "id": "31b14937-2524-491f-b0a0-dc0a7393ff33",
```

Comment:
Can we please provide real life examples, e.g. "Stuttgart" rather than "City", otherwise there is no point having examples really.